### PR TITLE
[DOCFIX] Update project Readme

### PR DESCRIPTION
**Pull request by apc999**
### What changes are proposed in this pull request?

Update Readme

### Why are the changes needed?

Various minor tunings on readme file

### Does this PR introduce any user facing changes?

None

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -89,38 +87,33 @@ to run a simple example with Alluxio.
...
   [Alluxio Java file system API](https://docs.alluxio.io/os/user/stable/en/api/FS-API.html#alluxio-java-api))
   to access all Alluxio-specific functionalities.
-- Artifact `alluxio-core-client-hdfs` provides a client for
+- Artifact `alluxio-core-client-hdfs` provides a Java artifact for
   [HDFS-Compatible file system API](https://docs.alluxio.io/os/user/stable/en/api/FS-API.html#hadoop-compatible-java-client).
-- Artifact `alluxio-shaded-client`, available in Alluxio 2.0.1 or later, includes both above
-  clients and all their dependencies but in a shaded form to prevent conflicts. This client jar is
-  self-contained but the size is also much larger than the other two. This artifact is recommended
-  generally for a project to use Alluxio client.
+- (Recommended) Artifact `alluxio-shaded-client` includes both artifact.
```
Comment (beinan) [inline README.md:102]: nit: move up `alluxio-shaded-client` as the first one of client artifacts?

**Review by beinan — APPROVED** [review]
Looks good to me

File: README.md (hunk 1 above)
Comment (apc999) [reply README.md]: done

**Comment by apc999** [comment]
alluxio-bot, merge this please

**Pull request merged**